src/main/kotlin/WorkerBuilder.kt

```diff
@@ -38,13 +44,14 @@ data class WorkerBuilder<T> private constructor(
     fun groupId(id: String) = option("group.id", id)
     fun option(k: String, v: Any) = options(mapOf(k to v))
     fun options(newOpts: Map<String, Any>) = copy(opts = opts + newOpts)
+    fun scope(scope: CoroutineScope): WorkerBuilder<T> = copy(scope = scope)
```

Comment:
Again, the name is vague and ambigious. Naming it coroutine scope or maybe even something that more reflects its a executing couroutine scope for the worker would be better.

Reply:
Sure!